intelmq/bots/parsers/shadowserver/_config.py

```diff
@@ -506,6 +506,52 @@ def convert_date_utc(value: str) -> Optional[str]:
     },
 }
 
+event4_microsoft_sinkhole_http = {
```

Comment:
What is the difference to the `event46_sinkhole_http` mapping?

Reply:
The same feed but now been send with two different names by shadowservers

Reply:
If just the name changed, we don't need an additional identical field-mapping, a change in `mapping` will do it.

Reply:
got it, will make the change